### PR TITLE
Readme Update

### PR DESCRIPTION
**Pull request by Rexogamer**

### CODE REVIEW

File: script.config [hunk 1]
```diff
@@ -3,3 +3,4 @@ localPath = C:\Projects\
 editor = code
 username = 
 password = 
+twoFacAuth = false
```
Comment (jarodburchill) [inline script.config]: @MisterWhopper is this needed?

File: README.md [hunk 2]
```diff
@@ -86,7 +92,7 @@ Create-react-app with TypeScript:
 ```
 react-ts
 ```
-Node.js project:
+Node.js with NPM:
```
Comment (jarodburchill) [inline README.md]: 'Node.js project' makes more sense

File: script.config (hunk 1 above)
Comment (MisterWhopper) [reply script.config]: We are going to need a way to tell if we need to do 2FA, but I think I can determine that at runtime. If not then I will add it in that PR

Comment (Rexogamer) [reply script.config]: my line of thought was that we’d have the tfa code in the main python file but only run it if this is true

Comment (jarodburchill) [reply script.config]: I'd like it to do a run-time check if that's possible

Comment (MisterWhopper) [reply script.config]: Yeah, but when the 2FA is needed it returns a really specific error message which we can detect. It's something like "Must specify two factor authentication one time password" or something like that

Comment (Rexogamer) [reply script.config]: I see

**Review by jarodburchill — COMMENTED** [review]
Okay, looks good!

**Review by jarodburchill — APPROVED** [review]

**Pull request merged**